src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/model/Constant.groovy

```diff
@@ -29,6 +29,11 @@ class Constant extends AbstractDescribableImpl<Constant> implements ExpandableCo
         this.value = StringUtils.trimToEmpty(value)
     }
 
+    Constant(Constant constant) {
```

Comment:
Couldn't we use the props and instantiate a new object instead of adding a new constructor
`new Constant(oldConst.getLabel(), olConst.getValue)`
Same for AdditionalSettings and *Configs